Sources/Clappr/Classes/Base/Loader.swift

```diff
@@ -1,63 +1,46 @@
 open class Loader {
-    internal(set) open var playbackPlugins: [Plugin.Type] = [AVFoundationPlayback.self, NoOpPlayback.self]
+    
+    static let shared = Loader()
+    
+    internal(set) open var playbacks: [Plugin.Type] = [AVFoundationPlayback.self, NoOpPlayback.self]
     #if os (iOS)
     internal(set) open var containerPlugins: [Plugin.Type] = [PosterPlugin.self, SpinnerPlugin.self]
     #else
     internal(set) open var containerPlugins: [Plugin.Type] = []
     #endif
     internal(set) open var corePlugins = [Plugin.Type]()
-
-    internal(set) open var mediaControl: MediaControl.Type = MediaControl.self
-
-    fileprivate var externalPlugins = [Plugin.Type]()
-
-    public convenience init() {
-        self.init(externalPlugins: [])
-    }
-
-    public init(externalPlugins: [Plugin.Type], options: Options = [:]) {
-        self.externalPlugins = externalPlugins
-
-        loadExternalMediaControl(options)
-
-        if !externalPlugins.isEmpty {
-            addExternalPlugins(externalPlugins)
-        }
-
+    
+    var externalPlugins: [Plugin.Type] = []
```

Comment:
Just to reiterate, let's not differentiate between registered and external plugins (there shouldn't be any hard-coded, built-in plugins).

The external plugins (passed via options) should only be used when loading the plugins into a context. The Loader instance shouldn't store those. This could cause plugins to leak from a Player instance to another.